cmd/git/branch.go

```diff
@@ -0,0 +1,76 @@
+package git
+
+import (
+  "fmt"
+  "io/ioutil"
+  "log"
+  "strings"
+
+  "github.com/Originate/gt/cmd/util"
+)
+
+
+func EnsureDoesNotHaveBranch(branchName string) {
+  if HasBranch(branchName) {
+    message := fmt.Sprintf("A branch named '%s' already exists", branchName)
+    util.ExitWithErrorMessage(message)
```

Comment:
`util.ExitWithErrorMessage` is for errors we catch. `log.fatal` is being used for unforeseen errors like filesystem errors.